docs/layers/eks/tutorials/how-to-setup-vanity-domains-on-alb-eks.mdx

```diff
@@ -58,7 +58,7 @@ duplicate certs for domains.
 
     #### Example
 
-    <Tabs groupId="eks-deployment-method">
+    <Tabs queryString="method" groupId="eks-deployment-method">
```

Comment:
method seems overly broad.

Maybe `software-delivery-method`

Reply:
let's use the same groupId, `eks-deployment-method`. I originally used method since I thought that was too long